### PR TITLE
Create Major Bowman in SSB

### PR DESCRIPTION
**Pull request by SSlayer9**

### CODE REVIEW

File: data/mods/ssb/moves.ts [hunk 1]
```diff
@@ -29,4 +29,29 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 	},
 	*/
 	// Please keep sets organized alphabetically based on staff member name!
+	"corrosivecloud": {
+		accuracy: 100, // a number or true for always hits
+		basePower: 100, // Not used for Status moves, base power of the move, number
+		category: "Physical", // "Physical", "Special", or "Status"
```
Comment (ghost) [inline data/mods/ssb/moves.ts]: set states that its special

File: data/mods/ssb/statuses.ts [hunk 2]
```diff
@@ -38,4 +38,17 @@ export const BattleStatuses: {[k: string]: ModdedPureEffectData} = {
...
+	majorbowman: {
+		noCopy: true,
+		onStart() {
+			this.add(`c|${getName('MajorBowman')}|Aaaand Cracktion!`);
+		},
+		onSwitchOut() {
+			this.add(`c|${getName('Username')}|This isn't Maury Povich!`);
+		},
+		onFaint() {
+			this.add(`c|${getName('Username')}|Never loved ya.`);
```
Comment (KrisXV) [inline data/mods/ssb/statuses.ts]:
```suggestion
			this.add(`c|${getName('MajorBowman')}|This isn't Maury Povich!`);
		},
		onFaint() {
			this.add(`c|${getName('MajorBowman')}|Never loved ya.`);
```

File: data/mods/ssb/random-teams.ts [hunk 3]
```diff
@@ -34,6 +34,12 @@ export const ssbSets: SSBSets = {
 	// Nature needs to be a valid nature with the first letter capitalized ex: Modest
 	*/
 	// Please keep sets organized alphabetically based on staff member name!
+	majorbowman: {
+		species: 'Weezing-Galar', ability: 'Neutralizing Gas', item: 'Black Sludge', gender: 'M',
+		moves: ['Strange Steam', ['Toxic Spikes', 'Haze'], 'Recover'],
+		signatureMove: 'Corrosive Cloud',
+		evs: {hp: 252, def: 252, spd: 4}, nature: 'Bold', level: 100, shiny: false,
```
Comment (KrisXV) [inline data/mods/ssb/random-teams.ts]:
```suggestion
		evs: {hp: 252, def: 252, spd: 4}, nature: 'Bold',
```

File: data/mods/ssb/moves.ts [hunk 4]
```diff
@@ -29,4 +29,29 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+			status: "brn",
+			chance: 30,
+		}, // secondary, set to null to not use one. Exact usage varies, check data/moves.js for examples
+		target: "normal",
+		type: "Poison",
+		onPrepareHit(target, source) {
+			this.add("-anim", source, "Poison Gas", target);
+			this.add("-anim", source, "Fire Spin", target);
+		},
+		onEffectiveness(typeMod, target, type) {
```
Comment (Instructing) [inline data/mods/ssb/moves.ts:52]: this isn't really how you do it. See how freeze dry works

File: data/mods/ssb/moves.ts [hunk 5]
```diff
@@ -29,4 +29,29 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+			if (type === 'Steel') return 1;
+		},
+		critRatio: 1,
```
Comment (ghost) [inline data/mods/ssb/moves.ts]: unnecessary

Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
		},
```

**Review by ghost — CHANGES_REQUESTED** [review]

File: data/mods/ssb/moves.ts [hunk 6]
```diff
@@ -29,4 +29,29 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+		category: "Physical", // "Physical", "Special", or "Status"
+		desc: "Has a 30% chance to burn the target, super effective against Steel.", // long description
+		shortDesc: "30% chance to burn, super effective against Steel", // short description, shows up in /dt
+		name: "Corrosive Cloud",
+		pp: 10,
+		priority: 0,
+		flags: {protect: 1, mirror: 1}, // Move flags https://github.com/smogon/pokemon-showdown/blob/master/data/moves.js#L1-L27
+		secondary: {
+			status: "brn",
+			chance: 30,
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
			chance: 30,
			status: 'brn',
```

File: data/mods/ssb/moves.ts [hunk 7]
```diff
@@ -29,4 +29,29 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+		}, // secondary, set to null to not use one. Exact usage varies, check data/moves.js for examples
+		target: "normal",
+		type: "Poison",
+		onPrepareHit(target, source) {
+			this.add("-anim", source, "Poison Gas", target);
+			this.add("-anim", source, "Fire Spin", target);
+		},
+		onEffectiveness(typeMod, target, type) {
+			if (type === 'Steel') return 1;
+		},
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
```

Move these below `flags`

File: data/mods/ssb/moves.ts [hunk 8]
```diff
@@ -29,4 +29,29 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+	"corrosivecloud": {
+		accuracy: 100, // a number or true for always hits
+		basePower: 100, // Not used for Status moves, base power of the move, number
+		category: "Physical", // "Physical", "Special", or "Status"
+		desc: "Has a 30% chance to burn the target, super effective against Steel.", // long description
+		shortDesc: "30% chance to burn, super effective against Steel", // short description, shows up in /dt
+		name: "Corrosive Cloud",
+		pp: 10,
+		priority: 0,
+		flags: {protect: 1, mirror: 1}, // Move flags https://github.com/smogon/pokemon-showdown/blob/master/data/moves.js#L1-L27
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
		flags: {protect: 1, mirror: 1}, // Move flags https://github.com/smogon/pokemon-showdown/blob/master/data/moves.js#L1-L27
		onPrepareHit(target, source) {
			this.add('-anim', source, 'Poison Gas', target);
			this.add('-anim', source, 'Fire Spin', target);
		},
		onEffectiveness(typeMod, target, type) {
			if (type === 'Steel') return 1;
		},
```

File: data/mods/ssb/moves.ts [hunk 9]
```diff
@@ -29,4 +29,29 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 	},
 	*/
 	// Please keep sets organized alphabetically based on staff member name!
+	"corrosivecloud": {
+		accuracy: true,
+		basePower: 90,
+		category: "Physical",
+		desc: "Has a 30% chance to burn the target, super effective against Steel.",
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
		desc: "Has a 30% chance to burn the target. This move's type effectiveness against Steel is changed to be super effective no matter what this move's type is.",
```

File: data/mods/ssb/moves.ts [hunk 10]
```diff
@@ -29,4 +29,29 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+		shortDesc: "30% chance to burn, super effective against Steel",
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
		shortDesc: "30% chance to burn. Super effective on Steel.",
```

**Comment by HoeenCoder** [comment]
I messed up the rebase and had to wipe it and re-make it

File: data/mods/ssb/random-teams.ts [hunk 11]
```diff
@@ -46,6 +46,12 @@ export const ssbSets: SSBSets = {
 		signatureMove: 'ebhewbnjgWEGAER',
 		evs: {spa: 252, spd: 4, spe: 252}, ivs: {atk: 0}, nature: 'Timid',
 	},
+	Majorbowman: {
```
Comment (ghost) [inline data/mods/ssb/random-teams.ts]: MajorBowman

**Pull request merged**